app/controllers/BaseController.java

```diff
@@ -67,31 +69,37 @@ public void setAuthenticationService(AuthenticationService authenticationService
     public void setCacheProvider(CacheProvider cacheProvider) {
         this.cacheProvider = cacheProvider;
     }
-    
+
     @Autowired
     public void setBridgeConfig(BridgeConfig bridgeConfig) {
         this.bridgeConfig = bridgeConfig;
     }
-    
+
     /**
      * Returns a session. Will not throw exception if user is not authorized or has not consented to research.
      * @return session if it exists, or null otherwise.
      */
-    protected UserSession getSessionIfItExists() {
+    UserSession getSessionIfItExists() {
         String sessionToken = getSessionToken();
```

Comment:
Just from looking at the code I wasn't sure why you were increasing the visibility of these to package scope. Needed?